Frontend/CO.CDP.OrganisationApp/Pages/Users/AddUser.cshtml

```diff
@@ -64,13 +65,13 @@
 
                 <div class="govuk-form-group @(emailHasError ? "govuk-form-group--error" : "")">
                     <label class="govuk-label" for="@nameof(Model.Email)">
-                        Email address
+                        @StaticTextResource.Users_EmailAddress_Label
                     </label>
 
                     @if (emailHasError)
                     {
                         <p id="email-error" class="govuk-error-message">
-                            <span class="govuk-visually-hidden">Error:</span>
+                            <span class="govuk-visually-hidden">@StaticTextResource.Global_ErrorColon</span>
```

Comment:
Can this be consistent with the other forms - This should be as follows, and we can remove the ErrorColon version 

```suggestion
                            <span class="govuk-visually-hidden">@StaticTextResource.Global_Error:</span>
```